.github/workflows/release.yml

```diff
@@ -89,6 +89,7 @@ jobs:
         uses: digicatapult/check-version@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          fail_on_same_version: false
```

Comment:
I'd prefer not to use this flag as it essentially disables check version. Instead let's either just merge this and cancel the release workflow or if you want you can bump the version manually.

We have another ticket which will fix this problem so we can just bump versions with a label but that's not there yet